staging/src/k8s.io/apiserver/pkg/endpoints/handlers/create.go

```diff
@@ -193,20 +194,24 @@ func createHandler(r rest.NamedCreater, scope *RequestScope, admit admission.Int
 				obj = scope.FieldManager.UpdateNoErrors(liveObj, obj, managerOrUserAgent(options.FieldManager, req.UserAgent()))
 				admit = fieldmanager.NewManagedFieldsValidatingAdmissionController(admit)
 			}
+			trace.Step("About to call mutatingAdmission")
 			if mutatingAdmission, ok := admit.(admission.MutationInterface); ok && mutatingAdmission.Handles(admission.Create) {
 				if err := mutatingAdmission.Admit(ctx, admissionAttributes, scope); err != nil {
 					return nil, err
 				}
 			}
+			trace.Step("mutatingAdmission.Admin finished")
 			// Dedup owner references again after mutating admission happens
 			dedupOwnerReferencesAndAddWarning(obj, req.Context(), true)
 			result, err := requestFunc()
+			trace.Step("Object stored in database", utiltrace.Field{"len", len(body)}, utiltrace.Field{"err", err})
 			// If the object wasn't committed to storage because it's serialized size was too large,
 			// it is safe to remove managedFields (which can be large) and try again.
 			if isTooLargeError(err) {
 				if accessor, accessorErr := meta.Accessor(obj); accessorErr == nil {
 					accessor.SetManagedFields(nil)
 					result, err = requestFunc()
+					trace.Step("Object stored in database, retry without managedFields", utiltrace.Field{"len", len(body)}, utiltrace.Field{"err", err})
```

Comment:
Same here - it doesn't mean it actually is stored.

Reply:
Done